### PR TITLE
Article fetch helper route

### PR DESCRIPTION
**Pull request by Eakanath1**
All routes are working fine

### CODE REVIEW

File: backend/controllers/user.js [hunk 1]
```diff
@@ -23,28 +33,48 @@ const getUserProfile = expressAsyncHandler(async (req, res) => {
...
+        user.languages = req.body.languages || user.languages;
+        user.experience = req.body.experience || user.experience;
+        user.education = req.body.education || user.education;
+        user.certifications = req.body.certifications || user.certifications;
+        user.achievements = req.body.achievements || user.achievements;
+        user.projects = req.body.projects || user.projects;
+        User.findByIdAndUpdate(
+          { _id: user._id }, { $set: user}, {new: true}, (err, userRes) => {
+            if (err) {
+            return res.status(400).res.json({
```
Comment (veerreshr) [inline backend/controllers/user.js:68]:
```
res.status(400);
throw new Error("Update Unsuccessful");
```
Change it in this way , as this is the convention followed for throwing errors

**Review by veerreshr — CHANGES_REQUESTED** [review]
Need to resolve merge conflict due to package-lock.json file and I added one change , have a look at it

File: backend/controllers/user.js (hunk 1 above)
Comment (Eakanath1) [reply backend/controllers/user.js]: Changes made. Check it now

Comment (veerreshr) [reply backend/controllers/user.js]: package-lock.json is still conflicting , once update your branch with latest changes from main

Comment (Eakanath1) [reply backend/controllers/user.js]: Conflicts are now resolved

**Review by veerreshr — APPROVED** [review]

**Pull request merged**